api/src/permissions/modules/process-ast/utils/validate-path/validate-path-existence.ts

```diff
@@ -0,0 +1,20 @@
+import type { SchemaOverview } from '@directus/types';
+import { createCollectionForbiddenError, createFieldsForbiddenError } from './create-error.js';
+
+export function validatePathExistence(path: string, collection: string, fields: Set<string>, schema: SchemaOverview) {
+	const collectionInfo = schema.collections[collection];
+
+	if (collectionInfo === undefined) {
+		throw createCollectionForbiddenError(path, collection);
+	}
+
+	const requestedFields = Array.from(fields);
+
+	const nonExistentFields = requestedFields.filter(
+		(field) => !field.startsWith('$FOLLOW') && collectionInfo.fields[field] === undefined,
+	);
```

Comment:
Fields such as `preferences_divider` do not exist in the DB, resulting in an error as it's non-existent.

For example in the app minimal perms:
https://github.com/directus/directus/blob/aafd63319ef354eeef8c92555c2ba88cdc8fb704/packages/system-data/src/fields/users.yaml#L65-L73

Reply:
Fixed in https://github.com/directus/directus/pull/22773/commits/ad6295310ef67c0999b3046f4e6580bb974f5b65